google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -1813,6 +1813,36 @@ public void testFastDDLQuery() throws InterruptedException {
     }
   }
 
+  @Test
+  public void testFastQuerySlowDDL() throws InterruptedException {
+    String tableName =
```

Comment:
Please also add a mock test showing that the subsequent getQueryResults call gets made.

Reply:
done